recipe/meta.yaml

```diff
@@ -19,21 +19,21 @@ source:
     - h5py_win_setup_build.py.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
     - cython
     - six
     - pkgconfig
 
   run:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
```

Comment:
If we had a libnetcdf feedstock, we would be able to move this up to ```1.8.16```, right?
In the meantime, we can pin to ```1.8.15``` IMO.

Work for you guys?